fmcg_store/data/pos_config.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="point_of_sale.pos_config_main" model="pos.config">
+        <field name="name">Grocery Shop</field>
+        <field name="discount_product_id" ref="point_of_sale.product_product_consumable"/>
+        <field name="is_margins_costs_accessible_to_every_user" eval="True"/>
+        <field name="iface_print_auto" eval="True"/>
+        <field name="ship_later" eval="True"/>
+        <field name="iface_available_categ_ids" eval="[Command.link(ref('pos_category_1')), Command.link(ref('pos_category_2')), Command.link(ref('pos_category_3')), Command.link(ref('pos_category_4')), Command.link(ref('pos_category_5')), Command.link(ref('pos_category_6')), Command.link(ref('pos_category_7')), Command.link(ref('pos_category_8')), Command.link(ref('pos_category_9'))]"/>
```

Comment:
This is only useful when trying to restrict categories, which is not the case here
```suggestion
```